crypto/bio/b_sock2.c

```diff
@@ -115,6 +115,58 @@ int BIO_connect(int sock, const BIO_ADDR *addr, int options)
     return 1;
 }
 
+/*-
+ * BIO_localaddr - Sets local address for socket
+ * @sock: the socket to set
+ * @addr: local address to bind to
+ * @options: BIO socket options
+ *
+ * Binds to the address using the given socket and options, then
+ * starts listening for incoming connections.
+ *
+ * Options can be a combination of the following:
+ * - BIO_SOCK_REUSEADDR: Try to reuse the address and port combination
+ *   for a recently closed port.
+ *
+ * When restarting the program it could be that the port is still in use.  If
+ * you set to BIO_SOCK_REUSEADDR option it will try to reuse the port anyway.
+ * It's recommended that you use this.
+ */
+
```

Comment:
Delete blank line between comment and function.
We tend not to put these kinds of length comments in the source, but rather in the manpage.  In fact, move this block comment into the appropriate pod page.  We don't want to add undocumented functions.

Reply:
Blank line deleted.  The comment style is exactly that used in the rest of crypto/bio/b_sock2.c